en/docs/get-started/overview.md

```diff
@@ -30,7 +30,7 @@ Basically, WSO2 Identity Server takes care of secure yet convenient sign-ins of
 
 ## Why WSO2 Identity Server?
 
-![IAM functionality](../../assets/img/guides/iam-functionality.png)
+![IAM functionality]({{base_path}}/assets/img/guides/iam-functionality.png)
```

Comment:
@himeshsiriwardana Shall I remove this file from the repo?
I don't think it is used anywhere?

Reply:
+1